apps/frontend/src/components/ProfileInfo/ProfileInfo.tsx

```diff
@@ -0,0 +1,41 @@
+import { UserWithRelatedData } from '@neighborhood/backend/src/types';
```

Comment:
I believe this should be `UserWithoutPasswordHash`.

Reply:
You're right. Will modify